experimental/perf/scripts/run_load_tests.py

```diff
@@ -24,16 +27,32 @@ def print_help():
 
 def run_test_case(test_case, log_folder):
     if test_case == "bench_log":
-        test_case_helper.bench_log(log_folder, 4, 10000)
+        loop_count = 4
+        step_count = 10000
+        logger.info(
+            f"Load testing SDK logging in {loop_count} iterations, "
+            + f"each logging {step_count} steps, 100 metrics and metric key size of 10"
+        )
```

Comment:
not required to add '+' i think if you have a set of () it should just work

Reply:
yep you are right!  I always confuse with syntax of shell scripting, java, etc.